api/src/pcapi/core/offerers/api.py

```diff
@@ -763,10 +763,10 @@ def delete_venue_banner(venue: models.Venue) -> None:
     search.async_index_venue_ids([venue.id])
 
 
-def can_offerer_create_educational_offer(offerer_id: int | None) -> None:
+def can_offerer_create_educational_offer(offerer_id: int) -> None:
     import pcapi.core.educational.adage_backends as adage_client
 
-    if offerer_id is None:
+    if settings.IS_INTEGRATION:
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏On est justement en train de supprimer ces vérifs sur l'env
L'idée est que l'api ne doit pas avoir besoin de savoir qu'on est en intégration

Là tu pourrais plutôt définir une variable d'env `CAN_OFFERER_CREATE_EDUCATIONAL_OFFER` par exemple et la mettre à 0 en environnement d'intégration

cf. https://github.com/pass-culture/pass-culture-main/pull/5334 pour exemple :smiley: